crate/crypto/src/crypto/symmetric/tests.rs

```diff
@@ -100,12 +128,84 @@ fn test_encrypt_decrypt_aes_xts_256() {
 
     let tweak = random_nonce(SymCipher::Aes256Xts).unwrap();
 
-    let (ciphertext, tag) = encrypt(SymCipher::Aes256Xts, &key, &tweak, &[], &message).unwrap();
+    let (ciphertext, tag) =
+        encrypt(SymCipher::Aes256Xts, &key, &tweak, &[], &message, None).unwrap();
     assert_eq!(ciphertext.len(), message.len());
     assert_eq!(tag.len(), AES_256_XTS_MAC_LENGTH); // always 0
 
-    let decrypted_data =
-        decrypt(SymCipher::Aes256Xts, &key, &tweak, &[], &ciphertext, &tag).unwrap();
+    let decrypted_data = decrypt(
+        SymCipher::Aes256Xts,
+        &key,
+        &tweak,
+        &[],
+        &ciphertext,
+        &tag,
+        None,
+    )
+    .unwrap();
+
+    // `to_vec()` conversion because of Zeroizing<>.
+    assert_eq!(decrypted_data.to_vec(), message);
+}
+
+#[test]
+fn test_encrypt_decrypt_aes_cbc_256_pkcs5_padding() {
+    // Load FIPS provider module from OpenSSL.
+    #[cfg(not(feature = "non-fips"))]
+    Provider::load(None, "fips").unwrap();
+
+    let mut message = vec![0_u8; 42];
+    rand_bytes(&mut message).unwrap();
+
+    let cipher = SymCipher::Aes256Cbc;
+    let key = random_key(cipher).unwrap();
+
+    let tweak = random_nonce(cipher).unwrap();
+
+    let (ciphertext, tag) = encrypt(cipher, &key, &tweak, &[], &message, None).unwrap();
+
+    let decrypted_data = decrypt(
+        SymCipher::Aes256Cbc,
+        &key,
+        &tweak,
+        &[],
+        &ciphertext,
+        &tag,
+        None,
+    )
+    .unwrap();
```

Comment:
```suggestion
    let (ciphertext, tag) = encrypt(cipher, &key, &tweak, &[], &message, Some(PaddingMethod::PKCS5)).unwrap();

    let decrypted_data = decrypt(
        SymCipher::Aes256Cbc,
        &key,
        &tweak,
        &[],
        &ciphertext,
        &tag,
        Some(PaddingMethod::PKCS5),
    )
    .unwrap();
```

From what I understand : this is the unit test responsible of testing the PaddingMethod PKCS5, however, no padding is used as opposed to the actual name of the test